src/megachatapi_impl.cpp

```diff
@@ -5136,21 +5182,35 @@ uint8_t MegaChatSessionPrivate::convertSessionState(uint8_t state)
 void MegaChatSessionPrivate::setState(uint8_t state)
 {
     this->state = convertSessionState(state);
+    changed |= MegaChatSession::CHANGE_TYPE_STATUS;
 }
 
 void MegaChatSessionPrivate::setAvFlags(AvFlags flags)
 {
     av = flags;
+    changed |= MegaChatSession::CHANGE_TYPE_REMOTE_AVFLAGS;
```

Comment:
Double-check this change is not reported for changes on our own AV flags, but only for remote (and viceversa)

Reply:
Checked.